BUILD.md

```diff
@@ -60,10 +60,23 @@ The following dependencies are required for this project:
     cmake --build build
     ```
 5. **Run the Agent**
-    To run the agent, copy the file `src/agent/service/wazuh-agent.service` to `/etc/systemd/system/`.
-    Set the environment variable WAZUH_HOME to the wazuh-agent executable directory.
-    You can start and stop the agent, and get status with:
+    To run the agent, copy the file `src/agent/service/wazuh-agent.service` to `/usr/lib/systemd/system/`.
```

Comment:
Can we still run the agent from the CLI without creating the service? If so, perhaps we can add two items here:
- Run the Agent from the CLI
- Run the Agent as a service

Or something similar.

Reply:
I think it would be a good idea to allow, at least in debug mode, for the agent to run directly from the cli without needing to install a service.

Reply:
Yes, this came up in conversation with @vikman90. I just added this to BUILD.md.
`--run` runs the agent in the foreground, with all output sent to stdout
`--start `starts the service